2024-10-14 14:45:29---DataSocketSystem.Net.Sockets.SocketException (0x80004005): 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。
   在 System.Net.Sockets.Socket.Receive(Byte[] buffer, Int32 offset, Int32 size, SocketFlags socketFlags)
   在 System.Net.Sockets.Socket.Receive(Byte[] buffer)
   在 RTC.ProcessFrm.DataSocket() 位置 D:\锐博特\机器人\stnli\RTC\ProcessFrm.cs:行号 2732
2024-10-14 14:45:29---DataSocketSystem.Net.Sockets.SocketException (0x80004005): 远程主机强迫关闭了一个现有的连接。
   在 System.Net.Sockets.Socket.Receive(Byte[] buffer, Int32 offset, Int32 size, SocketFlags socketFlags)
   在 System.Net.Sockets.Socket.Receive(Byte[] buffer)
   在 RTC.ProcessFrm.DataSocket() 位置 D:\锐博特\机器人\stnli\RTC\ProcessFrm.cs:行号 2732
